glean-core/rlb/src/core_metrics.rs

```diff
@@ -13,6 +13,9 @@ pub struct ClientInfoMetrics {
     pub app_display_version: String,
     /// The product-provided release channel (e.g. "beta").
     pub channel: Option<String>,
+    /// The locale of the application during initialization (e.g. "es-ES").
+    /// If the locale can't be determined on the system, the value is "und", to indicate "undetermined".
```

Comment:
That doesn't apply here though, does it? If this field is unset we don't set it in the client info.
The foreign language bindings set it to `und`.